.github/workflows/pr-assign.yml

```
@@ -59,22 +59,31 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - uses: ibexa-yuna/changed-files@v12-b
+      - name: Check PR files against mask
         id: changed-files
-        with:
-          sha: ${{ github.event.pull_request.head.sha }}
-          files: |
-            ${{ matrix.params.extension }}
+        shell: bash {0} # Disable fail-fast when no match is found
+        run: |
+          matches=$(hub api /repos/${REPO}/pulls/${PR}/files | jq -r '.[].filename | split("/") | .[-1]' | grep -Ec '${{ matrix.params.extension }}')
+          if [ "${matches}" -ge 1 ]; then
+            output=1
+          else
+            output=0
+          fi
+          echo "::set-output name=matches::$output"
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          REPO: ${{ github.repository }}
+          PR: ${{ github.event.number }}
 
       - name: Assign Pull Request maintainers
-        if: ${{ steps.changed-files.outputs.any_changed }}
+        if: ${{ steps.changed-files.outputs.matches }} == 1
```

```suggestion
        if: steps.changed-files.outputs.matches == 1
```
I think we can remove it, per doc:
```
When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. 
```
(https://docs.github.com/en/actions/learn-github-actions/expressions)

Otherwise it should look like: `if: ${{ steps.changed-files.outputs.matches == 1 }}`?
At least that's what I get when I look at the examples, see https://docs.github.com/en/actions/learn-github-actions/expressions#example-for-composite-action-step

---

I just prefer the ${{ }} syntax, so would like to keep it this way.